plugins/services/src/js/components/ServiceBreadcrumbs.js

```diff
@@ -114,36 +115,43 @@ class ServiceBreadcrumbs extends React.Component {
     return this.lastStatusWidth;
   }
 
-  getHealthStatus(service) {
+  getServiceStatus(service) {
     if (service == null || !this.state.shouldRenderServiceStatus) {
       return null;
     }
 
-    const serviceStatus = service.getStatus();
-    const runningTasksCount = service.getTaskCount();
-    const instancesCount = service.getInstancesCount();
+    let progressBar = null;
+    let iconDisplay = null;
+    const { tasksRunning } = service.getTasksSummary();
+    const instancesTotal = service.getInstancesCount();
+    const tooltipContent = `${tasksRunning} ${StringUtil.pluralize("instance", tasksRunning)} running out of ${instancesTotal}`;
 
-    if (instancesCount === 0) {
-      return null;
+    if (this.props.taskID == null && this.props.params != null) {
+      progressBar = (
+        <BreadcrumbSupplementalContent hasProgressBar={true}>
+          <ServiceStatusProgressBar key="status-bar" service={service} />
+        </BreadcrumbSupplementalContent>
+      );
     }
 
-    const taskCountDetails = runningTasksCount === instancesCount
-      ? `(${runningTasksCount})`
-      : `(${runningTasksCount} of ${instancesCount})`;
+    if (this.props.params != null) {
```

Comment:
the reason I check for `this.props.params != null` is because we do not want to display the icon and progress bar when it's not service detail